src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -91,6 +91,8 @@
 
 OPEN_PR_ISSUE_DESCRIPTION_LENGTH = 52
 
+MAX_RECENT_ISSUES = 10000
```

Comment:
This is _10,000_ now instead of _1000_, typo?

Reply:
this is erroring for some people because we are returning 50k+ groups for the `Group` query. i thought 10k would be reasonable because you might be modifying something that is erroring but may not be the most common error? 🤷‍♀️

Reply:
we have to be returning a LOT of groups from the query we're doing now because the error says `After processing, query is 884919 bytes, which is too long for ClickHouse to process. Max size is 262144 byte`

Reply:
We talked offline and I am caught up on this, makes sense!